Bookish/Views/Home/Index.cshtml

```diff
@@ -4,34 +4,80 @@
-            @Html.TextBoxFor(m => m.BookInput.BookName, new{@type="text", placeholder="Book name"})
-            @Html.TextBoxFor(m => m.BookInput.ISBN, new{@type="text", placeholder="ISBN", pattern="[0-9]{13}", title = "Please enter a 13 digit number"})
-            @Html.TextBoxFor(m => m.BookInput.Publisher, new{@type="text", placeholder="Publisher"})
-            @Html.TextBoxFor(m => m.BookInput.DatePublished, new{@type="date", placeholder="Publish date"})
-            @Html.TextBoxFor(m => m.BookInput.Author, new{@type="text", placeholder="Author", pattern="^[A-Za-z ,.'-]+$" })
-            @Html.TextBoxFor(m => m.BookInput.NewCopies, new{@type="number", placeholder="copies"})
-            <input type="submit" value="Enter book info" />
-
+            <table class="table table-hover table-sm">
+                
+                <tbody>
+                    <tr>
+                        <td><b>Book title : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.BookName, new { @type = "text", placeholder = "Book name", required = "required"})</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book ISBN: </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.ISBN, new { @type = "text", placeholder = "ISBN", pattern = "[0-9]{13}", title = "Please enter a 13 digit number", required = "required" })</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book publisher : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.Publisher, new { @type = "text", placeholder = "Publisher"})</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book publish date : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.DatePublished, new { @type = "date", placeholder = "Publish date", required = "required" })</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book author forename : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.AuthorForename, new { @type = "text", placeholder = "Forename", pattern = "^[A-Za-z ,.'-]+$"})</td>
+                    </tr>
+                    <tr>
+                        <td><b>Book author surname : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.AuthorSurname, new { @type = "text", placeholder = "Surname", pattern = "^[A-Za-z ,.'-]+$" })</td>
+                    </tr>
+                    <tr>
+                        <td><b>Number of copies : </b></td>
+                        <td>@Html.TextBoxFor(m => m.BookInput.NewCopies, new { @type = "number", placeholder = "copies", required = "required" })</td>
+                    </tr>
+                </tbody>
+            </table>
+            <br/><input type="submit" value="Enter book info"/>
         </fieldset>
     }
-    
+</div>
+<div id="Enter a User" class="tabcontent" style="display:none;">

```

Comment:
This is just here to hide one of the initial tabs displaying the results, wasn't sure if there was a nicer way to do this since the tabs work on pure css, I couldnt find a bootstrap method for them